sympy/geometry/line.py

```diff
@@ -28,7 +28,6 @@
 from sympy.functions.elementary.piecewise import Piecewise
 from sympy.functions.elementary.trigonometric import (_pi_coeff as pi_coeff, acos, tan, atan2)
 from sympy.geometry.exceptions import GeometryError
-from sympy.geometry.util import intersection
```

Comment:
Why removed? Quality tests automatically detect unused imports if they are present.

Reply:
Actually, it was showing errors in travis-ci build and there were suggestions to remove this statements. Should I keep this commit or remove?